betty/cropper/templates/image.js

```diff
@@ -33,6 +33,19 @@
     };
   }
 
+  // Credit to http://ejohn.org/blog/getboundingclientrect-is-awesome/
+  function isInViewport(el) {
```

Comment:
Worth re-using this somehow? https://github.com/theonion/bulbs-elements/blob/master/lib/bulbs-elements/util/in-view-monitor.js

/cc @collin